cql/ManageSpecialPopulation.cql

```diff
@@ -365,7 +365,7 @@ define RecommendationForPatientsYoungerThan25:
       Collate.HasNoTreatmentAfterDate(Collate.LastKnownCin2Biopsy.date)
     ) then
       {
-        short: 'See Details',
+        short: 'Surveillance 6-month follow-up',
         date: Today(),
         group: 'Younger Than 25 (K.1.4)',
         details: {
```

Comment:
Here Surveillance does not mean "HPV or cotesting" as elsewhere. Should review the recommendation text, and see if details needs updating. Should short be changed to something more specific in this case?